googleCalender.js

```diff
@@ -0,0 +1,427 @@
+const max = 50;
+async function usersFullSync({ dataStore, client }) {
+  const responseData = await client.fetch(
+    "admin/directory/v1/users?customer=my_customer&projection=full&viewType=admin_view"
+  );
+
+  if (!responseData.ok) {
+    throw new Error(
+      `Events sync failed ${responseData.status}:${responseData.statusText}.`
```

Comment:
Since this is users API call error message should be `Users`
```suggestion
      `Users sync failed ${responseData.status}:${responseData.statusText}.`
```